tox.ini

```diff
@@ -12,10 +12,10 @@ envlist =
     py
 
 [testenv]
-whitelist_externals = 
-    pytest
 commands =
-    pytest
+    python -m pytest
```

Comment:
Why this change?

Reply:
This keeps it uniform whether one runs `pytest` from the terminal or `python-m pytest` from within `tox`.